src/Lecture1.hs

```diff
@@ -119,4 +136,27 @@ and lower than 6 elements (4, 5, 6, 7, 8 and 9).
 
 🕯 HINT: Use recursion to implement this function.
 -}
-lowerAndGreater n list = error "TODO"
+lowerAndGreater :: Int -> [Int] -> String
+lowerAndGreater n list = makeText n numsLessAndMore{-(numLess, numMore)-}
+  where
+    makeText :: Int -> (Int, Int) -> String
+    makeText a (b, c) = unwords [
+      show a
+      , "is greater than"
+      , show b
+      , "elements and lower than"
+      , show c
+      , "elements"]
+    numsLessAndMore = count n list
```

Comment:
You don't need to pass `n` as an argument to `makeText` or `count`. Functions inside `where` can see arguments of the top-level function because they are in scope. So it can help to remove some extra arguments.

Reply:
updated